src/libtorch.cc

```diff
@@ -1155,19 +1222,75 @@ ModelInstanceState::ProcessRequests(
   }
 
   uint64_t compute_end_ns = 0;
+  if (Kind() == TRITONSERVER_INSTANCEGROUPKIND_GPU) {
+#ifdef TRITON_ENABLE_GPU
```

Comment:
Seems redundant to have it everywhere, can this be encapsulated somehow
```
  if (Kind() == TRITONSERVER_INSTANCEGROUPKIND_GPU) {
#ifdef TRITON_ENABLE_GPU
```

Reply:
Refactored.